src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain
+
+private const val MIN_CARS_SIZE = 2
+
+class Cars(
+    private val cars: List<Car>,
+) {
+    val value: List<Car>
+        get() = cars.toList()
```

Comment:
흐흐 이름이 겹쳐서 `value`로 네이밍을 하신 것 같군요!
요거는 아래와 같이 변경해서 많이 쓰는편 입니다!
```suggestion
    private val _cars: List<Car>,
) {
    val cars: List<Car>
        get() = _cars.toList()
```